Funky-Chickens/bangazonAPI
# created faker data for orders and training tables and ran add faker d…

###I made the following changes:  
- created faker data for orders and training tables and ran add faker data functions in build-db
- i did this so we would have data in these tables to test our API functionality


### Steps to test:
- run node build-db.js in git bash and open sqlite to see if all columns in orders and training tables were populated.

### feature ticket: #16
